Makefile

```diff
@@ -21,7 +21,11 @@ else
 endif
 
 .PHONY: check
-check: lint fmt vet verify-codegen test ## Run code validations
+check: lint fmt vet verify-codegen check-pkg test ## Run code validations
+
+.PHONY: check-pkg
+check-pkg:
+	dep check | grep -q cluster-api-actuator-pkg && exit 1 || exit 0
```

Comment:
Given we run only `make test-e2e` and `make k8s-e2e` in the CI, it's still useful to have this run by hand.

Reply:
This will also be run in CI, but as a separate optional job.